### ISSUE TITLE
Middle align the search icon when using custom `--font` and/or `--font-size` values

### ISSUE THREAD

**Issue by veselinvelichkov** [joeattardi/picmo#159]
In some cases, when we use a custom `font` or `font-size`, the search icon will not be middle aligned in its container. 

In the example preview below I have used the following configuration and noticed that the icon was not properly middle aligned.

```CSS
--font: "Open Sans", sans-serif;
--font-size: 14px;
````
![emoji-picker-before](https://user-images.githubusercontent.com/38430625/117949959-5a92ef80-b31b-11eb-882e-f8cddb9ec193.png)

We could easily fix that by setting a `line-hight: 1` in the styles of the search icon .
```SCSS
.emoji-picker__search-icon {
    line-height: 1;
}
````
**The result:**
![emoji-picker-after](https://user-images.githubusercontent.com/38430625/117950173-96c65000-b31b-11eb-96b4-47f38f0e1cd0.png)

I hope this issue will be addressed soon 🤞 
Thank you in advance @joeattardi 🙂

**Comment by joeattardi** [comment]
Nice catch! Feel free to submit a PR with this fix, otherwise I can incorporate the fix into v5.

**Comment by joeattardi** [comment]
This issue is resolved in `v5.0.0-beta.6`.

The icon size is no longer set via font-size, but with width and height. It is center aligned now at different sizes:

<img width="389" alt="image" src="https://user-images.githubusercontent.com/219285/166132249-d8eff395-4e79-489f-a9b9-caca7697f024.png">
